docs/source/configuration/overview.mdx

```diff
@@ -598,9 +598,13 @@ supergraph:
 
 </MinVersion>
 
-<ExperimentalFeature />
+<Note>
+
+Beginning with [v1.49.0](https://github.com/apollographql/router/releases/tag/v1.49.0), the router supported enhanced operation signature normalization [experimentally](/resources/product-launch-stages/#experimental-features).
+In [v.1.53.0](https://github.com/apollographql/router/releases/tag/v1.53.0), this configuration became [generally available](/resources/product-launch-stages/#general-availability).
```

Comment:
Suggest listing GA and Experimental versions to be more scannable. Also applies to other Note in PR.

```suggestion

The router supports enhanced operation signature normalization in the following versions:

* [General availability](/resources/product-launch-stages/#general-availability) in v1.53.0 and later
* [Experimental](/resources/product-launch-stages/#experimental-features) in v1.49.0 to v1.52

```